addons/project/models/res_partner.py

```diff
@@ -8,7 +8,9 @@
 class ResPartner(models.Model):
     """ Inherits partner and adds Tasks information in the partner form """
     _inherit = 'res.partner'
+    _check_company_auto = True
 
+    project_ids = fields.One2many('project.project', 'partner_id', string='Projects', check_company=True)
```

Comment:
I am not sure to understand why do we need to add this field? 🤔

Reply:
It is to trigger the check company once the company is changed on the partner. 🙂